twitter_inserter_posts/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.12 as builder
```

Comment:
After merging the Dockerfile cleanup, you can use `golang:1.13`, use just `/app` as a Workdir and don't have to prepend `GO111MODULE=on` everywhere